source/MaterialXGenOgsXml/OgsXmlGenerator.cpp

```diff
@@ -250,7 +250,7 @@ namespace
     }
 }
 
-const string OgsXmlGenerator::OUTPUT_NAME = "out";
+const string OgsXmlGenerator::OUTPUT_NAME = "outColor";
```

Comment:
It's not really a color all of the time, but if this makes it work guess we can leave it as this for now.

Reply:
Well, if it's a Maya surface shader DG node then it's expected to be a color. The type can definitely be arbitrary in other use cases, such when using the generated fragment with fragment graph API to construct an MShaderInstance directly.